src/main/java/com/box/sdk/BoxFolder.java

```diff
@@ -1264,6 +1264,27 @@ public boolean getCanNonOwnersInvite() {
             return this.canNonOwnersInvite;
         }
 
+        /**
+         * Sets whetherhether or not the non-owners can invite collaborators to the folder.
+         *
+         * @param canNonOwnersInvite indicates non-owners can invite collaborators to the folder.
+         */
+        public void setCanNonOwnersInvite(boolean canNonOwnersInvite){
+            this.canNonOwnersInvite = canNonOwnersInvite;
+            this.addPendingChange("can_non_owners_invite", canNonOwnersInvite);
+        }
+
+        /**
+         * Sets whether future collaborations should be restricted to within the enterprise only.
+         *
+         * @param isRestricted indicates whether there is collaboration restriction within enterprise.
+         */
+        public void setIsCollaborationRestrictedToEnterprise(boolean isRestricted) {
```

Comment:
CI is failing, I think because this method already exists: 
```
/home/travis/build/box/box-java-sdk/src/main/java/com/box/sdk/BoxFolder.java:1302: error: method setIsCollaborationRestrictedToEnterprise(boolean) is already defined in class BoxFolder.Info
        public void setIsCollaborationRestrictedToEnterprise(boolean isRestricted) {
                    ^
```

Could you remove it and see if that fixes things?

Reply:
sorry about that don't know how that got passed my test build :( 
removed duplicate code and also fixed typo in comments

Reply:
@mattwiller Any thoughts why CI is failing? Says some sort styling guide error